man/systemd.network.xml

```diff
@@ -2457,6 +2457,26 @@
         </listitem>
       </varlistentry>
 
+       <varlistentry>
+        <term><varname>CakeOverhead=</varname></term>
+        <listitem>
+          <para>Specifies the size of each packet in bytes on the wire may differ from that seen by Linux. Takes one of
+          <literal>ethernet</literal>, <literal>ethernet-vlan</literal> or ranges a number in the range -64 to 256.
+          Setting ethernet equivalent to overhead 38 mpu (rounds each packet including overhead up to a minimum length)
+          84 and setting ether-vlan adds 4 bytes to the overhead compensation, accounting for an IEEE 802.1Q VLAN header
+          appended to the Ethernet frame header. Defaults to unset and kernel's default is used.
```

Comment:
hmm, i really don't grok this. Can you carefully reword this? I am not sure how to fix this, as I don't get behaviour here myself?